www/js/lib/settingsStore.js

```diff
@@ -34,7 +34,7 @@ define([], function () {
    */
   var regexpCookieKeysToMigrate = new RegExp([
     'hideActiveContentWarning', 'showUIAnimations', 'appTheme', 'useCache',
-    'lastContentInjectionMode', 'listOfArchives', 'lastSelectedArchive'
+    'lastContentInjectionMode', 'listOfArchives', 'lastSelectedArchive', 'useHomeKeyToFocusSearchBar'
```

Comment:
It's not necessary to add your new setting here.
This was to provide a smooth migration of the settings found in old storage : this can not happen on new settings, that are always stored in the new storage